Orm/Xtensive.Orm/Orm/Logging/BaseLog.cs

```diff
@@ -29,30 +29,32 @@ public abstract class BaseLog
     /// <summary>
     /// Creates region. Within the region, all messages are indented.
     /// </summary>
-    /// <param name="message">Message, which writes to beginning of region and end of region.</param>
-    /// <param name="parameters">Values of parameters in <paramref name="message"/>.</param>
+    /// <param name="messageId">Identifier of message in resources that should be written at the begging and at the end of the region or message itself.</param>
```

Comment:
My last suggestion for this was wrong and it eliminated one of the params in the description. I'm fixing my mistake here
```suggestion
    /// <param name="messageId">Identifier of the message in resources that should be written at the begging and at the end of the region or message itself.</param>
    /// <param name="parameters">Values of parameters in <paramref name="messageId"/> (resource string or message itself).</param>
```